packages/blade/src/_helpers/storybook/StoryPageWrapper.tsx

```diff
@@ -49,15 +55,15 @@ const StoryPageWrapper = (props: StoryPageWrapperTypes): React.ReactElement => {
       {figmaURL !== '#' ? (
         <FigmaEmbed src={figmaURL} title={`${props.componentName} Figma Designs`} />
       ) : null}
-      {props.children}
+      <BladeProvider themeTokens={paymentTheme}>{props.children}</BladeProvider>
       {props.imports === '' ? null : (
         <>
           <Title>Imports</Title>
-          <Highlight language="tsx">
+          <SandboxHighlighter>
```

Comment:
New highlighter in the town 🥳  It maintains the same theme as sandpack examples

## Before
<img width="799" alt="image" src="https://user-images.githubusercontent.com/30949385/222672793-248f57a8-8882-47d6-9236-0beda593138a.png">


## After
<img width="775" alt="image" src="https://user-images.githubusercontent.com/30949385/222672680-d4d3511d-6454-4cb6-9e66-965bba2364ba.png">

Reply:
Oh it looks like a bug? I removed it for aesthetics 😆

Reply:
this should look better lol?
<img width="678" alt="image" src="https://user-images.githubusercontent.com/30949385/225004850-62cb9842-6d71-4d2c-add2-6658089d0205.png">